presto-postgresql/src/test/java/com/facebook/presto/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -201,6 +229,15 @@ private void testUnsupportedDataType(String databaseDataType)
         }
     }
 
+    private static DataType<byte[]> byteaDataType()
+    {
+        return DataType.dataType(
+                "bytea",
+                VARBINARY,
+                bytes -> format("bytea E'\\\\x%s'", base16().encode(requireNonNull(bytes, "bytes is null"))),
```

Comment:
No need for `requireNonNull` here as the `encode` method will throw NPE.

The rationale for null checks in constructors and other places is to ensure the object invariants and to have the exception occur at the right time, when you know who provided the null. In this case, the caller will get an NPE with the proper stack trace.

Occasionally, we might want the explicit check to provide a better message, but that shouldn't be needed here as there's only one object.

Now, if `bytes` was an argument to this method and was being put directly into the lambda capture, we'd want a null check to ensure the capture is valid.